react/features/base/tracks/functions.js

```diff
@@ -33,16 +34,16 @@ export function createLocalTracksF(
     let { cameraDeviceId, micDeviceId } = options;
 
     if (typeof APP !== 'undefined') {
-        // TODO The app's settings should go in the redux store and then the
-        // reliance on the global variable APP will go away.
+        store || (store = APP.store); // eslint-disable-line no-param-reassign
```

Comment:
why do we need this? have the caller pass the store, the jsdoc doesn't say it's optional.

Reply:
Ah, I see. Then please leave the comment so we remember!